### PR TITLE
Remove presentation of app-specific webhook URLs.

### PR DESCRIPTION
**Pull request by bbewan**
https://app.clubhouse.io/buddybuild/story/6924/update-all-webhook-docs-to-update-the-example-webhook-url-we-no-longer-include-the-appid

Also includes some formtting improvements.

### CODE REVIEW

File: repository/kiln/webhook.adoc [hunk 1]
```diff
@@ -14,52 +14,52 @@ your testers have the very latest changes.
...
 
-In the **Repositories** field, start typing the name of your repository,
-then select it. Remember that the buddybuild webhook URL is specific to
-your app, so you must create a new web hook in Kiln for each app, even
-if your repository contains multiple apps.
-
+. In the **Repositories** field, start typing the name of your repository,
+then select it.
++
+Remember that the buddybuild webhook URL is specific to your app, so you
```
Comment (Palleas) [inline repository/kiln/webhook.adoc]: I'm not sure if that's true anymore. I think the URL is now unique and buddybuild finds the associated apps when it receives a notification. 
* Summons @richiebres for confirmation *

**Comment by bbdocs** [comment]
See the website here: http://153-bb-docs.surge.sh

**Comment by bbdocs** [comment]
See the website here: http://153-bb-docs.surge.sh

File: repository/kiln/webhook.adoc (hunk 1 above)
Comment (bbewan) [reply repository/kiln/webhook.adoc]: @richiebres When you get a chance, can you confirm the @Palleas comment?

**Comment by bbdocs** [comment]
See the website here: http://153-bb-docs.surge.sh

File: repository/kiln/webhook.adoc (hunk 1 above)
Comment (richiebres) [reply repository/kiln/webhook.adoc]: This is true for GitHub, Bitbucket, GitLab. Not true for the others.
So the text you have is good Ewan.

Regardless, the webhook URL shown in the dialog is *always* correct.

Comment (bbewan) [reply repository/kiln/webhook.adoc]: Thanks @richiebres!

**Comment by bbdocs** [comment]
See the website here: http://153-bb-docs.surge.sh

**Pull request merged**